tests/src/test/scala/common/Wsk.scala

```diff
@@ -869,7 +869,8 @@ class WskApi()
             { apiname map { a => Seq("--apiname", a) } getOrElse Seq() } ++
             { swagger map { s => Seq("--config-file", s) } getOrElse Seq() } ++
             { responsetype map { t => Seq("--response-type", t) } getOrElse Seq() }
-        cli(wp.overrides ++ params, expectedExitCode, showCmd = true, env=Map("WSK_CONFIG_FILE" -> cliCfgFile.getOrElse("")))
+        val envMap = { cliCfgFile map {f => Map("WSK_CONFIG_FILE" -> f) } getOrElse Map() }
```

Comment:
Also it seems dangerous to me to drive these tests with an external setting in this way; because again, I can run the tests and forget that I have this value set and have funky behavior. The point of these tests to integrate against a specific deployment via wsk.properties. So what settings are you changing externally that you can't do with whisk.properties?